pkg/arvo/sys/vane/ames.hoon

```diff
@@ -2222,7 +2255,8 @@
       =/  =message-pump-state
         (~(gut by snd.peer-state) bone *message-pump-state)
       ::
-      =/  message-pump    (make-message-pump message-pump-state channel)
+      =/  message-pump
```

Comment:
It doesn't look like we need to thread peers.ames-state into this core anymore.

Reply:
GOOD point